packages/flutter_tools/lib/src/commands/config.dart

```diff
@@ -116,6 +116,13 @@ class ConfigCommand extends FlutterCommand {
       // We send the analytics event *before* toggling the flag intentionally
       // to be sure that opt-out events are sent correctly.
       AnalyticsConfigEvent(enabled: value).send();
+      if (!value) {
+        // Normally, we wait for the analytics to all send before the tool
```

Comment:
nit: instead of "we", document the part of the tool that is waiting normal.

Reply:
Done.